wolnut/__init__.py

```diff
@@ -1,7 +1,18 @@
-import logging
+__title__ = 'wolnut'
+__author__ = 'hardwarehaven'
+__license__ = 'MIT'
+__copyright__ = 'Copyright 2025 hardwarehaven'
+__version__ = '1.0.0'
```

Comment:
These attributes are unnecessary. They just duplicate information avaiable from `importlib.metadata`. e.g. see https://packaging.python.org/en/latest/discussions/single-source-version/

Reply:
Actually you are reading the versoin from here, but the other attributes are duplicates.

Reply:
You're right. Will be removing it.